cli/crates/server/src/udf_builder.rs

```diff
@@ -160,6 +161,11 @@ pub async fn build(
     let new_package_json_contents = serde_json::to_string_pretty(&package_json).expect("must be valid JSON");
     trace!("new package.json contents:\n{new_package_json_contents}");
 
+    // symlink to grafbase-wasm-sdk
```

Comment:
would have been worth saying why instead, the function is explicit enough for what it does IMHO. Basically what you say in the PR. But since git blame can be used, good enough.^^